packages/extension-chakra-storefront/src/components/product-tile/index.jsx

```diff
@@ -17,7 +17,7 @@ import {
     Skeleton as ChakraSkeleton,
     Text,
     Stack,
-    useMultiStyleConfig,
+    useSlotRecipe,
```

Comment:
Nit, let's try to get these components in alphanetic order, and the hooks import will usually go last. See example. See related [comment](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2425#discussion_r2089440795)